grails-app/conf/UrlMappings.groovy

```diff
@@ -609,6 +609,11 @@ class UrlMappings {
             action = [GET: "statusOptions"]
         }
 
+        "/api/purchaseOrders/rollbackOrder/$id"(parseRequest: true) {
```

Comment:
Endpoint should be: `/api/purchaseOrders/$id/rollback` and action should be called `rollback`